### PR TITLE
Install ntfsprogs if WE is available as registered product

### PR DESCRIPTION
**Pull request by mimi1vx**
http://argus.suse.cz/tests/885
http://argus.suse.cz/tests/884

### CODE REVIEW

**Comment by sysrich** [comment]
Why?

File: tests/kernel/install_ltp.pm [hunk 1]
```diff
@@ -22,21 +22,36 @@ sub add_repos {
...
-sub try_add_workstation_addon {
-    my $have_wse = get_var('BUILD_WE') && get_var('ISO_2');
-    if ($have_wse) {
+sub scc_we_enabled {
+    script_run("zypper -n products -i", 0);
+    my $ret = wait_serial(qr/Workstation Extension/);
+    return $ret ? 1 : 0;
+}
+
+sub we_avivable {
```
Comment (okurz) [inline tests/kernel/install_ltp.pm]: do you mean available?

**Comment by mimi1vx** [comment]
for this https://progress.opensuse.org/issues/15652

File: tests/kernel/install_ltp.pm [hunk 2]
```diff
@@ -22,21 +22,36 @@ sub add_repos {
     zypper_call('--gpg-auto-import-keys ref');
 }
 
-sub try_add_workstation_addon {
-    my $have_wse = get_var('BUILD_WE') && get_var('ISO_2');
-    if ($have_wse) {
+sub scc_we_enabled {
+    script_run("zypper -n products -i", 0);
+    my $ret = wait_serial(qr/Workstation Extension/);
+    return $ret ? 1 : 0;
```
Comment (okurz) [inline tests/kernel/install_ltp.pm]: just `return wait_serial(…);` should be enough

Comment (mimi1vx) [inline tests/kernel/install_ltp.pm]: thx, good idea

File: tests/kernel/install_ltp.pm (hunk 1 above)
Comment (mimi1vx) [inline tests/kernel/install_ltp.pm]: yes, I always miswrite this word. Thanks

**Comment by mimi1vx** [comment]
This extend preparation of test image to install ntfsprogs if is one
condition true:

1) is DISTRI = opensue
2) is defined BUILD_WE and ISO_2
3) is installed WE product on SUT

extends poo#15652

**Comment by okurz** [comment]
+1

File: tests/kernel/install_ltp.pm [hunk 3]
```diff
@@ -22,21 +22,35 @@ sub add_repos {
     zypper_call('--gpg-auto-import-keys ref');
 }
 
-sub try_add_workstation_addon {
-    my $have_wse = get_var('BUILD_WE') && get_var('ISO_2');
-    if ($have_wse) {
+sub scc_we_enabled {
+    script_run("zypper -n products -i", 0);
+    return wait_serial(qr/Workstation Extension/);
```
Comment (richiejp) [inline tests/kernel/install_ltp.pm]: timeout as discussed

Comment (mimi1vx) [inline tests/kernel/install_ltp.pm]: unfortuanetly default value is safe , calling `zypper products` refreshes repositories and services which takes time and 5 sec is too short

**Pull request merged**